test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetRestoreTests.cs

```diff
@@ -1500,6 +1512,38 @@ public void Dotnet_New_Template_Restore_Success(string template)
             }
         }
 
+        [PlatformTheory(Platform.Linux)]
+        [InlineData("nunit-test")]
+        [InlineData("razorcomponent")]
+        [InlineData("page")]
+        [InlineData("viewimports")]
+        [InlineData("viewstart")]
+        [InlineData("gitignore")]
+        [InlineData("globaljson")]
+        [InlineData("nugetconfig")]
+        [InlineData("tool-manifest")]
+        [InlineData("webconfig")]
+        [InlineData("sln")]
+        [InlineData("proto")]
+        public void Dotnet_New_Template_CreateItem_Success(string template)
```

Comment:
Can you explain why we're adding testcases for templates that don't involve NuGet?

Reply:
Ok. Removed.